### ISSUE TITLE
vue/max-attributes-per-line insuccess

### ISSUE THREAD

**Issue by F-ckAllINeedIsU** [vuejs/eslint-plugin-vue#1074]
**Tell us about your environment**	
* **ESLint version:^6.7.2**	
* **eslint-plugin-vue version:^6.1.2**	
* **Node version:10.16.3**	

**Please show your full configuration:**

```js
module.exports = {
  root: true,
  env: {
    node: true
  },
  'extends': [
    'plugin:vue/essential',
    'eslint:recommended',
    '@vue/typescript/recommended'
  ],
  parserOptions: {
    ecmaVersion: 2020
  },
  rules: {
    'vue/max-attributes-per-line': [2, {
      'singleline': 1,
      'multiline': {
        'max': 1,
        'allowFirstLine': true
      }
    }]
  }
}

```

**What did you do?**	

```vue
<template>
  <div id="app">
    <img alt="Vue logo" src="./assets/logo.png">
  </div>
</template>
```

**What did you expect to happen?**
the second attributes need be next line and keep indent
```vue
<template>
  <div id="app">
    <img alt="Vue logo"
         src="./assets/logo.png">
  </div>
</template>
```

**What actually happened?**

the second attributes is on next line but not keep indent

```vue
<template>
  <div id="app">
    <img alt="Vue logo"
src="./assets/logo.png">
  </div>
</template>
```

**Comment by Landen13** [comment]
I've faced the same issue, @F-ckAllINeedIsU . How did you fix it?

**Comment by F-ckAllINeedIsU** [comment]
> I've faced the same issue, @F-ckAllINeedIsU . How did you fix it?

add rule

    'vue/html-indent': ['error', 2, {
      'attribute': 2,
      'baseIndent': 1,
      'closeBracket': 0,
      'alignAttributesVertically': true,
      'ignores': []
    }],

**Comment by Landen13** [comment]
@F-ckAllINeedIsU didn't work for me
also i have prettier rules added

**Comment by F-ckAllINeedIsU** [comment]
prettier do not allow attributes-per-line
see prettier official website about html lint


| |
chenye461209730
|
|
邮箱：chenye461209730@163.com
|

Signature is customized by Netease Mail Master

On 03/18/2020 20:57, Bogdan wrote:

@F-ckAllINeedIsU didn't work for me
also i have prettier rules added
